starter-code/css/main.css

```diff
@@ -6,3 +6,158 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+.nav {
+    background-color: white;
+    width: 100vw;
+    height: 60px;
+    top: 12px;
+    position: fixed;
+}
+.nav img {
+    margin-top: -4px;
+    height: 50px;
```

Comment:
when adjusting image sizes, you usually only manipulate the width or the height. Controlling the width auto adjust the height and vise versa, Unless you need a specific height and width but many times that distorts the image from what it should look like.